crm_location/tests/test_crm_location.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2017 Tecnativa - Luis M. Ontalba
+# Copyright 2019 Tecnativa - Alexandre Díaz
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0
+
+from odoo.tests import common
+
+
+class TestCrmLocation(common.SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestCrmLocation, cls).setUpClass()
+        cls.country = cls.env["res.country"].create({"name": "Test country"})
```

Comment:
```suggestion
        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
        cls.country = cls.env["res.country"].create({"name": "Test country"})
```
BTW can't we simply use the existing records?

Reply:
@simahawk Why do you use `tracking_disable=True` in the context ? Please suggest me.

can't we simply use the existing records? >> Do you mean use the existing records replace of creating new record ?

Reply:
fixed.

Reply:
because here we create of records that are probably not required and this add a lot of writes which slow down tests (consider we have tons of tests in all repos).
We could reuse existing demo data and change only the fields we need it.
Using tracking_disable makes sure we shut down tracking machinery which is useless and brings a lot of not required overhead.